os/arch/arm/src/artik053/src/artik053_tash.c

```diff
@@ -272,6 +272,7 @@ int board_app_initialize(void)
 	struct mtd_dev_s *mtd;
 #endif /* CONFIG_RAMMTD */
 
+#ifndef CONFIG_DISABLE_MOUNTPOINT
```

Comment:
1. If you want to add this, you should move this to above part of variable definitions.
2. They (artik053_configure_partition, CONFIG_ARTIK053_AUTOMOUNT_USERFS, so on) already have a dependancy with DISABLE_MOUNTPOINT. Do we need to add an addtitional conditional? I don't think so.